py-polars/tests/unit/datatypes/test_struct.py

```diff
@@ -970,3 +970,11 @@ def test_struct_out_nullability_from_arrow() -> None:
 def test_empty_struct_raise() -> None:
     with pytest.raises(ValueError):
         pl.struct()
+
+
+def test_named_exprs() -> None:
+    df = pl.DataFrame({"a": 1})
+    schema = {"b": pl.Int64}
+    res = df.select(pl.struct(schema=schema, b=pl.col("a")))
```

Comment:
`res` and `expect` seem to be the exact same query. Can you check the valid outputs via a `assert res.to_dict(as_series=False) ==  expected`

and a `res.schema == expected_schema`?

Reply:
Sure, I made that part more direct. Thanks @ritchie46 .